src/Engines/QueryBuilderEngine.php

```diff
@@ -340,10 +341,13 @@ protected function compileRelationSearch($query, $relation, $column, $keyword)
                 $relationMethod = "whereRaw";
             }
 
+            //overwrite the first element of the binding - we expect the keyword to be located there
+            $bindings[0]  = $this->prepareKeyword($keyword);
```

Comment:
Just tested the PR and it seems like this line is not need and can be removed. Will merge and do the necessary changes. Thanks for the hint!